e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs

```diff
@@ -25,6 +25,9 @@ public class IIoTPlatformTestContext : IDisposable, IDeviceConfig, IIoTHubConfig
             Configuration = GetConfiguration();
 
             RegistryHelper = new RegistryHelper(IoTHubConfig);
+            EdgeBaseDeployment = new IoTHubEdgeBaseDeployment(this);
+            PublisherDeployment = new IoTHubEdgeBaseDeployment(this);
```

Comment:
Those two (`EdgeBaseDeployment` and `PublisherDeployment`) are specific to standalone tests and not the whole test suite. So please move them to `PublishNodesFromFileTestTheory` class instead.

On a side note, using `this` for initiating `IoTHubEdgeBaseDeployment` and `IoTHubEdgeBaseDeployment` here doesn't look right to me. We are still in the constructor of `IIoTPlatformTestContext` object, so the object itself cannot be considered fully initialized just yet. So using it to initialize something else doesn't look correct to me. Happy to discuss this further.

Reply:
OK, after trying to implement this change myself I realized that not passing `this` from within `IIoTPlatformTestContext` to constructors requires a lot of wrangling and is not really straightforward. So the second part of the comment can be ignored.

Reply:
Addressed in #955.